src/printer.rs

```diff
@@ -274,3 +290,68 @@ fn color_of_outcome(outcome: &Outcome) -> ColorSpec {
     out.set_fg(Some(color));
     out
 }
+
+#[cfg(test)]
+mod tests {
+    use std::io::Cursor;
+
+    use super::*;
+
+    #[test]
+    fn list_ignored() {
+        let tests = vec![
+            Test {
+                name: "foo".into(),
+                kind: "".into(),
+                is_ignored: false,
+                is_bench: false,
+                data: (),
+            },
+            Test {
+                name: "bar".into(),
+                kind: "bar-test-kind".into(),
+                is_ignored: true,
+                is_bench: false,
+                data: (),
+            },
+            Test {
+                name: "baz".into(),
+                kind: "baz-test-kind".into(),
+                is_ignored: false,
+                is_bench: true,
+                data: (),
+            },
+            Test {
+                name: "quux".into(),
+                kind: "".into(),
+                is_ignored: true,
+                is_bench: true,
+                data: (),
+            },
+        ];
+
+        {
+            // Without --ignored (all tests)
+            let output: Vec<u8> = Vec::new();
+            let mut cursor = Cursor::new(output);
```

Comment:
`Vec<u8>` already implements `Write`, even without `Cursor`. So the cursor should not be necessary here, correct?